_docs_explore/64-request-items.md

```diff
@@ -1,4 +1,277 @@
+| `@type`   | "CreateAttributeRequestItem"                                                                                                                                                 | Specifies the type of the Request item for parsing.                                                                                                                                              |
+| attribute | [IdentityAttribute]({% link _docs_explore/61-data-model.md %}#identityattribute) \| [RelationshipAttribute]({% link _docs_explore/61-data-model.md %}#relationshipattribute) | The IdentityAttribute or RelationshipAttribute to create for the peer within the Identity of the peer.<br>The owner of the Attribute which should be created can only be the recipient Identity. |
+
+### Response
+
+#### Item Properties
+
+| Name        | Type                                | Description                      |
+| ----------- | ----------------------------------- | -------------------------------- |
+| `@type`     | "CreateAttributeAcceptResponseItem" | The type of the ResponseItem.    |
+| attributeId | `string`                            | The id of the created Attribute. |
+
+### Combinations and usage scenarios
+
+| Attribute Type | Attribute Owner | Possible? | Automation      | Examples/Reason                                                                                                                                                                                                                                                       |
+| -------------- | --------------- | --------- | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Identity       | Sender          | N         |                 | Use ShareAttributeRequestItem instead.                                                                                                                                                                                                                                |
+| Identity       | Recipient       | Y         | `USER_DECISION` | University sends student his certificate (Propose would be inappropriate in this case, because the student should not be able to return his own value)                                                                                                                |
+| Identity       | `<empty>`       | Y         |                 | An empty owner defaults to an Attribute with owner=`<recipient>`. This is needed for Requests inside of Relationship Templates, since you don’t know the Enmeshed Address of your peer before the Relationship is established.                                        |
```

Comment:
Hier müsste in der Automation Spalte derselbe Wert stehen wie bei "Attribute Owner = Recipient"